modules/core/src/bitgo.ts

```diff
@@ -2001,7 +1994,7 @@ export class BitGo {
    * Create a url for calling BitGo microservice APIs
    */
```

Comment:
This should be marked as `@deprecated` now, but we can deal with that in a follow up

Reply:
actually we still need to use this to wrap the url with baseUrl prefix.